docs-core/src/main/java/com/sismics/docs/core/model/jpa/Review.java

```diff
@@ -0,0 +1,102 @@
+package com.sismics.docs.core.model.jpa;
+
+import java.util.Date;
+
+import javax.persistence.Column;
+import javax.persistence.Entity;
+import javax.persistence.Id;
+import javax.persistence.Table;
+
+import com.google.common.base.MoreObjects;
+
+/**
+ * Link between an user and a group.
```

Comment:
I don't really understand what the function does based on this description. Do you think you could explain it in depth such that a person that is not very familiar with the codebase could understand it better?